### PR TITLE
 BaSP-MR-24: Super Admin - Tests for Get, Post, Put and Delete methods  

### PR DESCRIPTION
**Pull request by Bigzufi**
- Adding seeds
- Get and Get by ID methods tests
- Post method tests
- Put method tests
- Delete method tests

### CODE REVIEW

File: jest.setup.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ beforeAll(async () => {
   };
 
   await mongoose.connect(uri, mongoOptions);
-});
+}, 4000);
```
Comment (maurojjzz) [inline jest.setup.js:19]: the default timeout is 5000 so i think you should remove the `, 4000` and just keep the `});` on line 19

**Review by maurojjzz — CHANGES_REQUESTED** [review]
i just requested a change on jest.setup.js because it'll be better only change that file if its really necessary

**Comment by Bigzufi** [comment]
Requested change fixed

**Review by maurojjzz — APPROVED** [review]
Good work

**Review by BareaMartin — APPROVED** [review]

**Pull request merged**